modules/core/src/config.ts

```diff
@@ -161,6 +168,20 @@ const formattedEosTokens = coins.reduce((acc: EosTokenConfig[], coin) => {
   return acc;
 }, []);
 
+const formattedAvaxCTokens = coins.reduce((acc: AvaxcTokenConfig[], coin) => {
```

Comment:
This construct is equivalent to `coins.filter().map()` which is perhaps more explicit
```javascript
const formattedAvaxCTokens = coins.filter(coin) => coin instanceof AvaxERC20Token).map((coin) => ({
      type: coin.name,
      coin: coin.network.type === NetworkType.MAINNET ? 'avaxc' : 'tavaxc',
      network: coin.network.type === NetworkType.MAINNET ? 'Mainnet' : 'Testnet',
      name: coin.fullName,
      tokenContractAddress: coin.contractAddress.toString().toLowerCase(),
      decimalPlaces: coin.decimalPlaces,
    });
```

Reply:
It's a good propose. but  it's copy-paste code. Do we change all of them?

Reply:
small improvements

Reply:
I'm fine with this but if you do it please do it as a separate commit or separate PR entirely.